### PR TITLE
Data import

### PR DESCRIPTION
**Pull request by samsawan**
Sorry I missed Friday, totally slipped my mind :(

### CODE REVIEW

**Comment by vsalvino** [comment]
All good, that's why it's nice to have 2 shifts per day. I'm seeing a couple duplicates in the diff, could you please check that out, and also hand-check a couple of the restaurant names that seem to be people entering multiple restaurant names?

File: site/data/data.json [hunk 1]
```diff
@@ -10825,6 +10905,20 @@
       "Joeys Italian Grille- Madison"
     ]
   },
+  {
+    "name": "Joeys Italian Grille (Madison)",
```
Comment (samsawan) [inline site/data/data.json]: Fixing this entry

File: site/data/data.json [hunk 2]
```diff
@@ -13648,6 +13822,21 @@
       }
     ]
   },
+  {
+    "name": "Merry Arts, Lakewood Village Tavern, Academy Tavern",
```
Comment (samsawan) [inline site/data/data.json]: @vsalvino is this a suspect entry? I hid it, added carl to the Lakewood village entry

File: site/data/data.json [hunk 3]
```diff
@@ -14804,6 +14993,20 @@
       }
     ]
   },
+  {
+    "name": "Oishi Asian Restaurant",
```
Comment (samsawan) [inline site/data/data.json]: Duplicate, removing

File: site/data/data.json [hunk 4]
```diff
@@ -15959,6 +16162,20 @@
       }
     ]
   },
+  {
+    "name": "Parma Pit Stop",
```
Comment (samsawan) [inline site/data/data.json]: Duplicate, removing

File: site/data/data.json [hunk 5]
```diff
@@ -19664,6 +19964,21 @@
       }
     ]
   },
+  {
+    "name": "Spotted Owl And L'Abatros",
```
Comment (samsawan) [inline site/data/data.json]: I hid this entry, added Kindra to the Spotted Owl entry

**Review by samsawan — COMMENTED** [review]
pushing another commit up shortly, i think this addresses the dupes

File: site/data/data.json (hunk 2 above)
Comment (vsalvino) [reply site/data/data.json]: Some people try to list multiple restaurants, even though we ask them not to. I usually just take the first one and chop off the rest.

File: site/data/data.json (hunk 5 above)
Comment (vsalvino) [reply site/data/data.json]: Perfect. You can actually delete it all together, the script won't re-add the same person regardless of what their place name is.

**Comment by vsalvino** [comment]
Sorry for all the feedback, appreciate the help. The Sunday peeps must have been having a hard time following instructions on the sign-up form 😅

**Comment by samsawan** [comment]
No problem @vsalvino, just removed the multi entries as well. I didn't think the import script would recognize the fact that I had deleted them from the CSV and try to re-add the multi entries when it was run again. I was mistaken, so hell yeah, no need to keep that data there with a 'hide' flag. 👍

**Comment by vsalvino** [comment]
Yea, the importing is a mess and super hacky. There definitely has to be a better way... although probably would require making an actual signup page versus a simple google form.

**Pull request merged**